airbyte-integrations/connectors/source-slack/source_slack/schemas/channel_messages.json

```diff
@@ -38,8 +38,7 @@
           "type": ["null", "string"]
         },
         "updated": {
-          "type": ["null", "string"],
-          "format": "date-time"
+          "type": ["null", "string"]
```

Comment:
Does normalization perform migration to the destination table when a type changes?

Reply:
No, you need to update the schema to get the latest data type.